Rockerfile

```diff
@@ -47,12 +45,12 @@ ENV PATH=/opt/rocker/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin
 
 CMD ["/opt/rocker/bin/rocker"]
 
-PUSH dockerhub.grammarly.io/rocker:{{ .Version }}
+{{ if $version }}
```

Comment:
Also, this condition does not make sense because you always assign a value to the `$version` variable.